mne/io/neuralynx/neuralynx.py

```diff
@@ -38,6 +39,31 @@ def read_raw_neuralynx(
     See Also
     --------
     mne.io.Raw : Documentation of attributes and methods of RawNeuralynx.
+
+    Notes
+    -----
+    Neuralynx files are read from disk using the Neo package (http://neuralensemble.org/neo/).
+    Currently, only reading of the ``.ncs files`` is supported.
+
+    ``raw.info["meas_date"]`` is read from the ``recording_opened`` property
+    of the first ``.ncs`` file (i.e. channel) in the dataset (a warning is issued
+    if files have different dates of acquisition).
+
+    Channel-specific high and lowpass frequencies of online filters are determined
+    based on the ``DspLowCutFrequency`` and ``DspHighCutFrequency`` header fields,
+    respectively. If no filters were used for a channel, the default lowpass is set
+    to the Nyquist frequency and the default highpass is set to 0.
+    If channels have different high/low cutoffs, ``raw.info["highpass"]`` and
+    ``raw.info["lowpass"]`` are then set to the maximum highpass and minimumlowpass
+    values across channels, respectively.
+
+    Other header variables can be inspected using Neo directly. For example::
+
+        from neo.io import NeuralynxIO
+        fname = 'path/to/your/data'
+        nlx_reader = NeuralynxIO(dirname=fname)
+        print(nlx_reader.header)
+        print(nlx_reader.file_headers.items())
```

Comment:
Pretty sure we'll need these to make `pytest` happy
```suggestion
            from neo.io import NeuralynxIO  # doctest: +SKIP
            fname = 'path/to/your/data'  # doctest: +SKIP
            nlx_reader = NeuralynxIO(dirname=fname)  # doctest: +SKIP
            print(nlx_reader.header)  # doctest: +SKIP
            print(nlx_reader.file_headers.items())  # doctest: +SKIP
```
You'll also need to

Reply:
Thanks! Added both in.